StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/FileHeaderAnalyzers.cs

```diff
@@ -25,7 +25,7 @@
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public class FileHeaderAnalyzers : DiagnosticAnalyzer
     {
-        private const string SA1633Identifier = "SA1633";
```

Comment:
:bulb: This change can be reverted. Reference `SA1633DescriptorMissing.Id` or `SA1633DescriptorMalformed.Id` instead.